deploy/vagrant/conf/ec2.yml

```diff
@@ -69,9 +74,18 @@ Block_Device_Mapping: []
 # Some Instance Type must need PV/HVM:
 #   http://aws.amazon.com/amazon-linux-ami/instance-type-matrix/
 # PV
-AMI: "ami-b06a98d8"
+# AMI: "ami-b06a98d8"
 # HVM
-# AMI: "ami-5b697332"
+AMI: "ami-5b697332"
+#
+# NOT ALL AMI supports Spot Request
+# This Amazone Linux AMI(http://aws.amazon.com/amazon-linux-ami/) in us-east-1 
+# is HVM and supports Spot Request.
+# (http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html)
+# To find AMI that supports Spot Request in other region, 
+# click "Spot Requests" on left panel in AWS EC2 console 
+# -> click "Request Spot Instances", then you can find the AMI you want.
+# AMI: "ami-1ecae776"
```

Comment:
The HVM that does not support spot instance is RHEL, the one that supports is Amazon Linux since RHEL does not support spot instance. 

Yes, we can default to Amazon Linux, what do you think?